client/src/components/NewTicket/index.jsx

```diff
@@ -53,3 +56,23 @@ export default class extends Component {
     );
   }
 }
+class NewTicket extends Component {
+  render() {
+    return (
+      <React.Fragment>
+        <MainSidebar selected="tickets" />
+        <Navbar
+          className="new-ticket-nav"
+          children={[
```

Comment:
@Anies12  You can pass props.children as though they were natural children in the markup, like this:
```
<Navbar className="new-ticket__nav">
  <div className="new-ticket__header">
    <h3>new ticket</h3>
  </div>
</Navbar>